cognite/neat/workflows/steps/lib/graph_extractor.py

```diff
@@ -413,15 +414,162 @@ def convert_json_to_triples(
                 else:
                     data = Literal(str(data))
                 property_counter += 1
-                graph.add((parent_node, URIRef(ns + property_name), data))
+                graph.graph.add((parent_node, URIRef(ns + property_name), data))
 
         # Start conversion with a root node
         root_node = URIRef(ns + "root")
-        graph.add((root_node, URIRef(ns + "label"), Literal("root node")))
-        graph.add((root_node, RDF.type, URIRef(ns + "root_node_id")))
+        graph.graph.add((root_node, URIRef(ns + "label"), Literal("root node")))
+        graph.graph.add((root_node, RDF.type, URIRef(ns + "root_node_id")))
         convert_json_to_triples(json_data, root_node, "root", "root", None)
-
+        graph_store.graph.restart()  # restarting the graph to release the memory
         return FlowMessage(
             output_text=f"Data from source file imported successfully. Imported {nodes_counter} objects \
                             and {property_counter} properties ."
         )
+
+
+class InstancesFromAvevaPiAF(Step):
```

Comment:
Suggest adding a simple test for this one

Reply:
Yes will do in separate PR (along with test for JSON importer) , as this requires to anonymize the data dump I have (can't be public) and it's a bit of work.